lib/workos/sso.rb

```diff
@@ -119,6 +119,36 @@ def profile(code:, project_id:)
         WorkOS::Profile.new(response.body)
       end
 
+      # Promote a DraftConnection created via IdP Link Embed such that the
+      # Enterprise users can begin signing into your application.
+      #
+      # @param [String] token The draft connection token that's been provided to
+      # you by the IdP Link Embed
+      #
+      # @example
+      #   WorkOS::SSO.promote_draft_connection(
+      #     token: 'draft_conn_429u59js',
+      #   )
+      #   => true
+      #
+      # @return [Bool] - returns `true` if successful, `false` otherwise.
+      # @see https://github.com/workos-inc/idp-link
+      sig do
+        params(
+          token: String,
+        ).returns(T::Boolean)
+      end
+      def promote_draft_connection(token:)
+        request = bearer_post_request(
+          path: '/draft_connections/convert',
```

Comment:
https://github.com/workos-inc/workos/pull/2288

Changed it to `/draft_connections/activate`